src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -0,0 +1,20 @@
+package wolox.training.controllers;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Controller
+public class BookController {
+  @Controller
```

Comment:
Hi! It's not necessary to create a inner Greeting class 😊

Reply:
Got it! I've pushed the changes :)